apps/explorer/src/api/assets/assetsApi.ts

```diff
@@ -109,5 +110,17 @@ export const assetsQueries = createQueriesCollection({
         return tokenBalances
       }
     })
+  },
+  prices: {
+    assetPrice: (coinGeckoTokenId: string, currency = 'usd') => ({
+      queryKey: ['assetPrice', coinGeckoTokenId, currency],
+      queryFn: async (): Promise<number> => {
+        const res = (await (
+          await fetch(`https://api.coingecko.com/api/v3/simple/price?ids=${coinGeckoTokenId}&vs_currencies=${currency}`)
+        ).json()) as AssetPriceResponse
+
+        return res[coinGeckoTokenId][currency]
+      }
+    })
```

Comment:
Interesting to see the implementation using tanstack! Just FYI, here's how it'd be done with rtkQuery: https://github.com/alephium/alephium-frontend/blob/master/apps/desktop-wallet/src/storage/assets/priceApiSlice.ts#L37-L47

I like how you made sure that we can query other tokens apart from ALPH.